Source/WTF/wtf/text/StringImpl.h

```diff
@@ -930,18 +931,22 @@ inline StringImpl::StringImpl(unsigned length)
     STRING_STATS_ADD_16BIT_STRING(m_length);
 }
 
-template<typename Malloc>
-inline StringImpl::StringImpl(MallocSpan<LChar, Malloc> characters)
-    : StringImplShape(s_refCountIncrement, { static_cast<const LChar*>(nullptr), characters.span().size() }, s_hashFlag8BitBuffer | StringNormal | BufferOwned)
+template<typename CharacterType, typename Malloc>
+std::span<const CharacterType> StringImpl::ensureStringImplMallocSpan(MallocSpan<CharacterType, Malloc>&& characters)
```

Comment:
We usually use `ensure` to indicate that we're doing one-time initialization of a persistent value. I would call this `toStringImplMallocSpan`.